var/spack/repos/builtin/packages/rocm-openmp-extras/package.py

```diff
@@ -180,7 +195,7 @@ def patch(self):
 
         # If not in a git repo the STRIP command will have an empty
         # argument. This is fixed in later versions.
-        if self.spec.version == Version('4.3.0'):
+        if self.spec.version >= Version('4.3.0'):
```

Comment:
If the strip is fixed in later versions, why is the check a greater than?

Reply:
Unfortunately, the small patch was not promoted to the release I originally thought. I expect this change to be in 5.0 and this condition will be updated once released.